src/test/java/com/thebois/models/world/WorldTests.java

```diff
@@ -177,4 +165,96 @@ private boolean matrixEquals(final ITile[][] worldMatrix1, final ITile[][] world
         return true;
     }
 
+    private static Stream<Arguments> getCorrectCoordinatesToTest() {
+        return Stream.of(Arguments.of(0, 0, 2, 2),
+                         Arguments.of(5, 5, 10, 10),
+                         Arguments.of(0, 0, 0, 0),
+                         Arguments.of(20, 20, 11, 11),
+                         Arguments.of(49, 49, 40, 40)
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("getCorrectCoordinatesToTest")
+    public void testFindNearestStructure(final int startX,
```

Comment:
No point in having `test` in the name of a test. It's already apparent that it is in fact a test.